core/src/empty/empty.module.css

```diff
@@ -2,13 +2,8 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    font-family: Inter;
 }
 
-.errorText {
-    font-style: normal;
-    font-weight: normal;
-    font-size: 14px;
-    line-height: 20px;
+.messageText {
     padding: 1rem;
```

Comment:
We don't use "rem", we use exact pixel unit

Reply:
And the code formatting here is still off. Should run prettier against your CSS files too

Reply:
<img width="1138" alt="Screen Shot 2021-03-24 at 12 41 08" src="https://user-images.githubusercontent.com/46236031/112261480-8b9d5f00-8c9e-11eb-8cef-ab0ebfec7021.png">

I already used Prettier to format the code.

Reply:
Hm that looks good then 👍 I mean the output in "OUTPUT" tab is correct